ext/mysql2/client.c

```diff
@@ -1086,6 +1086,41 @@ static VALUE rb_mysql_client_encoding(VALUE self) {
 #endif
 
 /* call-seq:
+ *    client.automatic_close?
+ *
+ * @return [Boolean]
+ */
+static VALUE get_automatic_close(VALUE self) {
+  GET_CLIENT(self);
+  return wrapper->automatic_close ? Qtrue : Qfalse;
+}
+
+/* call-seq:
+ *    client.automatic_close = false
+ *
+ * Set this to +false+ to leave the connection open after it is garbage
+ * collected. To avoid "Aborted connection" errors on the server, explicitly
+ * call +close+ when the connection is no longer needed.
+ *
+ * @see http://dev.mysql.com/doc/en/communication-errors.html
+ */
+static VALUE set_automatic_close(VALUE self, VALUE value) {
+  GET_CLIENT(self);
+  if (RTEST(value)) {
+    value = Qtrue;
```

Comment:
I would avoid re-using the variable `value`, since we're not modifying the argument and shouldn't give the impression that we might be trying to do that.